tests/unitTests/material_model_tests/test_material_holzapfel_ogden_MA.cpp

```diff
@@ -0,0 +1,499 @@
+    }
+}
+
+// Test order of convergence between finite difference PK2 stress and compute_pk2cc() PK2 stress for random F (large)
+TEST_F(USTRUCT_HolzapfelOgdenMATest, TestPK2StressConvergenceOrderRandomFLarge) {
+    //verbose = true; // Show order of convergence, errors, F, S
+
+    // Loop over F in F_large_list
+    for (auto F_std : F_large_list) {
+        // Convert to Array
+        convertToArray(F_std, F);
+
+        // Check order of convergence between finite difference and compute_pk2cc() PK2 stress
+        TestHO_ma->testPK2StressConvergenceOrder(F, delta_max, delta_min, order, convergence_order_tol, verbose);
+    }
+}
+
+// Test order of convergence of consistency of material elasticity for triaxial stretch
+TEST_F(USTRUCT_HolzapfelOgdenMATest, TestMaterialElasticityConsistencyConvergenceOrderTriaxialStretch) {
+    //verbose = true; // Show order of convergence, errors, F, S
+
+    // Create a deformation gradient F for triaxial stretch
+    Array<double> F = {{1.1, 0.0, 0.0},
+                        {0.0, 1.2, 0.0},
+                        {0.0, 0.0, 1.3}};
+
+    // Check order of convergence of consistency of material elasticity
+    TestHO_ma->testMaterialElasticityConsistencyConvergenceOrder(F, delta_max, delta_min, order, convergence_order_tol, verbose);
+}
+
+// Test order of convergence between finite difference PK2 stress and compute_pk2cc() PK2 stress for triaxial compression
+TEST_F(USTRUCT_HolzapfelOgdenMATest, TestMaterialElasticityConsistencyConvergenceOrderTriaxialCompression) {
+    //verbose = true; // Show order of convergence, errors, F, S
+
+    // Create a deformation gradient F for triaxial compression
+    Array<double> F = {{0.9, 0.0, 0.0},
+                        {0.0, 0.8, 0.0},
+                        {0.0, 0.0, 0.7}};
+
+    // Check order of convergence of consistency of material elasticity
+    TestHO_ma->testMaterialElasticityConsistencyConvergenceOrder(F, delta_max, delta_min, order, convergence_order_tol, verbose);
+}
+
+// Test order of convergence between finite difference PK2 stress and compute_pk2cc() PK2 stress for biaxial stretch/compression
+TEST_F(USTRUCT_HolzapfelOgdenMATest, TestMaterialElasticityConsistencyConvergenceOrderBiaxialStretchCompression) {
+    //verbose = true; // Show order of convergence, errors, F, S
+
+    // Create a deformation gradient F for biaxial stretch/compression
+    Array<double> F = {{1.2, 0.0, 0.0},
+                        {0.0, 0.8, 0.0},
+                        {0.0, 0.0, 1.0}};
+
+    // Check order of convergence of consistency of material elasticity
+    TestHO_ma->testMaterialElasticityConsistencyConvergenceOrder(F, delta_max, delta_min, order, convergence_order_tol, verbose);
+}
+
+// Test order of convergence of consistency of material elasticity for random F (small)
+TEST_F(USTRUCT_HolzapfelOgdenMATest, TestMaterialElasticityConsistencyConvergenceOrderRandomFSmall) {
+    //verbose = true; // Show order of convergence, errors, F, S
```

Comment:
A lot of these functions have the same structure and are repeated all over.

Reply:
@aabrown100-git I've not used Google Test before so I don't know how tests can be organized. 

If each material model has essentially the same test then that would suggest using templates (somehow!) and being able to perform a test on a list of fixtures I guess.

You can think about this as what new code is required when adding (1) a new material model or (2) a new test. 

But it all works so we can probably think about this another time.